Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -192,32 +199,32 @@ export default function widgetBehavior(publicAPI, model) {
         handle1.getActive()
       ) {
         model.representations[0].setHandleVisibility(true);
-        model.handle1ToHide = true;
+        handleToHide[0] = true;
```

Comment:
declare that variable locally in `toggleHandleVisibility`

Reply:
the issue about this is that i will loose track of handles to hide when i exit the scope of this function

Reply:
We will see the usefulness of this variable once the work on the mixin is done